src/components/Sidebar/Sidebar.js

```diff
@@ -0,0 +1,57 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+import styled, { css } from 'react-emotion';
+
+import Header from './components/Header';
+import NavList from './components/NavList';
+import NavItem from './components/NavItem';
+import Backdrop from './components/Backdrop';
+import CloseButton from './components/CloseButton';
+
+const baseStyles = ({ theme, open }) => css`
+  label: sidebar;
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+  width: 256px;
```

Comment:
What do you think about this value of `256px` become a constant variable?

Reply:
This makes total sense! Thanks for pointing it out! 😄